code/modules/uplink/uplink_items.dm

```diff
@@ -1355,13 +1355,28 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/reverse_bear_trap
 	restricted_roles = list("Clown")
 
+/datum/uplink_item/role_restricted/clumsyDNA
+	name = "Clumsy Clown DNA"
+	desc = "A DNA injector that has been loaded with the clown gene that makes people clumsy. \
```

Comment:
desc = "A DNA injector that has been loaded with the clown gene that makes people clumsy.